tensorflow/lite/tools/versioning/op_version.cc

```diff
@@ -287,15 +291,26 @@ int GetBuiltinOperatorVersion(const OpSignature& op_sig) {
       }
       return 1;
 
+    case BuiltinOperator_MAXIMUM:
+    case BuiltinOperator_MINIMUM:
+    case BuiltinOperator_PACK:
+      if (op_sig.input_types.at(0) == TensorType_INT8) {
```

Comment:
I would avoid grouping pack w/ MAX/MIN, OK to have 2 separate case branches.

Reply:
@jdduke done